### DISCUSSION TITLE
Which programs are essential?

### DISCUSSION THREAD

**Discussion by ikspekt** [seerge/g-helper#581] [Q&A] [answered]
<img width="514" alt="스크린샷 2023-06-09 162154" src="https://github.com/seerge/g-helper/assets/136056477/792c3eeb-12b6-4b78-9244-df3371693547">
<img width="513" alt="스크린샷 2023-06-09 162207" src="https://github.com/seerge/g-helper/assets/136056477/03b42111-aa82-4a25-a03a-9f388265c6c3">

I would like to reduce asus programs to minimum.

I know that some programs are essential to ghelper, but I wonder which ones are essential.

Thank you.

**Comment by Metalogic999** [comment]
debloat.bat stops the non-essential Asus services from running and in pre-release build 0.81 there is a button in G-Helper to stop/start these services.

**Comment by seerge** [comment, marked as answer]
@ikspekt hello, i **wouldn't advice deleting anything** from system folders unless you really know and understand what you do.

Even g-helper's own "bloat/debloat.bat" (which is now available as a button in the app itself), just STOPS services, but don't delete them. So you can always go back.

**Reply by Metalogic999** [reply]
Agreed! I haven't even uninstalled the MyAsus app, in case I need to use it for support, I've just stopped the unnecessary services form running

**Reply by ikspekt** [reply]
I deleted some like the picture above, but the gpu fan doesn't spin.
Looks like I should reinstall all,, ;)
By any chance, are you willing to make it work on its own without the asus program later?

**Reply by seerge** [reply]
@ikspekt i really doubt that gpu fan is controlled by any of those. You fans are controlled by bios on low level (as they work w/o windows at all)
